config/tag-check.sh

```diff
@@ -23,14 +23,14 @@ fi
 # image.
 
 # Since we're in a PR, and there may be multiple commits, we want to
-# check all of them; so compare against the last merge commit before
-# this one. This should work both locally (that should correspond to
-# upstream/master) and in CI (which creates a merge commit to test
-# against).
-# NOTE: This assumes we always merge with merge commits.
-last_merge_commit=$(git log --merges -n1 HEAD^ --format=%H)
-if [[ -n "$(git diff $last_merge_commit --name-only config/)" ]]; then
+# check all of them; so compare against the fork point of this branch.
+# Don't compare
```

Comment:
truncated comment?

Reply:
🤦 yes. I'll tackle in a follow up as I don't want to repush the image/tag if that's okay?

Reply:
absolutely